### PR TITLE
learn/accessibility/typography Directly link Google Fonts

### PR DESCRIPTION
**Pull request by davelab6**
Issue: #8982

Changes proposed in this pull request:

- Directly link to Google Fonts

### CODE REVIEW

**Comment by heyawhite** [comment]
@cehfisher any concerns? Is Google Foundries fonts somehow different from Google Fonts?

File: src/site/content/en/learn/accessibility/typography/index.md [hunk 1]
```diff
@@ -60,8 +60,8 @@ tight. Pay special attention to kerning, especially between the problematic
 letter pair r/n. Otherwise, words like "yarn" could change to "yam" or "stern"
 to "stem," entirely changing the meaning of the copy. 
 
-Accessible typeface collections like the
-[Google Foundry on Adobe Fonts](https://fonts.adobe.com/foundries/google) can
+Open-source typeface collections like 
+[Google Fonts](https://fonts.google.com) can
```
Comment (heyawhite) [inline src/site/content/en/learn/accessibility/typography/index.md:64]:
```suggestion
Open source typeface collections like [Google Fonts](https://fonts.google.com) can
```

**Review by heyawhite — APPROVED** [review]

**Comment by cehfisher** [comment]
@heyawhite - The Google Foundry link is used for accessible fonts embedded in design products that use font libraries, like Adobe Illustrator. The Google Fonts link is for web-based products like websites, which can link out to the fonts directly. Since we have both designers and coders in the audience, I think it would make sense to include both resources.

File: src/site/content/en/learn/accessibility/typography/index.md [hunk 2]
```diff
@@ -60,8 +60,7 @@ tight. Pay special attention to kerning, especially between the problematic
 letter pair r/n. Otherwise, words like "yarn" could change to "yam" or "stern"
 to "stem," entirely changing the meaning of the copy. 
 
-Accessible typeface collections like the
-[Google Foundry on Adobe Fonts](https://fonts.adobe.com/foundries/google) can
+Open source typeface collections like [Google Fonts](https://fonts.google.com) can
 aid you in selecting the most inclusive typeface for your next design.
```
Comment (heyawhite) [inline src/site/content/en/learn/accessibility/typography/index.md:64]:
```suggestion
Open source typeface collections like [Google Fonts](https://fonts.google.com) can
aid you in selecting the most inclusive typeface for your next design.
This collection can also be found in [Adobe Fonts foundries](https://fonts.adobe.com/foundries/google).
```

**Comment by cehfisher** [comment]
```
Open-source typeface collections like [Google Fonts](https://fonts.google.com) can aid you in selecting the most inclusive typeface for your next digital or design-focused project.

If you use Adobe products, you can embed accessible font families from partnering foundry collections directly into your design, which includes select [Google Fonts](https://fonts.adobe.com/foundries/google) as well.
```

I made some mods to the copy to illustrate that you can also download Google fonts for your local design-focused projects, as I am not sure my last response captured that point.

Also, the Google Font foundry on Adobe only includes 103 font families, while the web-based collection has 1455 font families, so the copy mods also reflect that point.

Let me know what you think @heyawhite

**Comment by davelab6** [comment]
@heyawhite @cehfisher THANK YOU!!!

**Comment by cehfisher** [comment]
Thanks for the suggestion @davelab6!

**Pull request merged**